config/kernel-monitor.json

```diff
@@ -58,6 +58,12 @@
 			"reason": "IOError",
 			"pattern": "Buffer I/O error .*"
 		},
+		{
+			"type": "temporary",
+			"condition": "MemoryError",
```

Comment:
"condition" is added whenever the problem is permanent. You can remove the condition.